src/sections/Projects/SMP/index.js

```diff
@@ -1,19 +1,19 @@
 import React from "react";
+import { StaticImage } from "gatsby-plugin-image";
 
 import Layout from "../../../components/layout"; 
-
 import SMPWrapper from "./smp.style";
 import { Row, Col, Container } from "../../../reusecore/Layout";
 import Button from "../../../reusecore/Button";
 import JoinCommunity from "../../Community/Join-community";
 
-import example from "./service-mesh-performance-specification-example.gif";
-import c_icon from "./c-icon.svg";
-import wasm_capacity from "./native-and-wasm-at-capacity-100rps.png";
-import latency from "./latency-at-scale.png";
-import client_capacity from "./client-capacity.png";
-import smp from "./smp-white.svg";
-import callout from "./callout.png";
+const example = "./service-mesh-performance-specification-example.gif";
+const callout = "../../../assets/images/callout/callout.png";
+const smp = "./smp-white.svg";
```

Comment:
I think this image is already available in `/assets`. If this is duplicate, please delete this and import from the assets one. Check it out once though.

Reply:
Yes I checked on the assets folder, could not find the exact photo but I will check again.